contact.html

```diff
@@ -46,22 +46,27 @@ <h2 class="has-color-red-500">Security</h2>
                     <a href="mailto:support@codidact.org">support@codidact.org</a>, mentioning SECURITY in the subject
                     line.
                 </p>
-
-		<h2>GitHub</h2>
-		<p>Codidact is an open-source project.  If you find a bug or are requesting a change, you're welcome to report an issue in <a href="https://github.com/codidact/qpixel">our primary repository</a>.  You can also use Meta (next option).  Don't worry about choosing between GitHub and Meta; we'll make sure issues and discussions end up in the right places.
-		</p>
+		    
                 <h2>Meta</h2>
                 <p>
                     If you're asking about how our sites or the organisation operates, or if you've got a feature request or a
                     bug report, <a href="https://meta.codidact.com/">Meta</a> is the place to do that. Post there with the
-                    details, and either the community will be able to answer or our staff can get back to you.  (It's ok to use GitHub if you prefer.  If something comes up on GitHub that the community should discuss, we'll bring it to Meta.)
+                    details, and either the community will be able to answer or our staff can get back to you.
                 </p>
+
+		<h2>GitHub</h2>
+		<p>Codidact is an open-source project. As an alternative to Meta, if you find a bug or are requesting a change, you're welcome to open an issue in <a href="https://github.com/codidact/qpixel">our primary repository</a>.  You can also use Meta.  Don't worry about choosing between GitHub and Meta; we'll make sure issues and discussions end up in the right places.
+		</p>
+
+		<h2>Chat</h2>
+		<p>We also have two chat servers! You are welcome to join our <a href="https://discord.gg/bv2aaGa">community chat server</a>, and alternatively, our <a href="https://discord.gg/7jf8UzS">development server</a> if you're interested in the development of Codidact.</p>
```

Comment:
Suggestion: ...join on community server, where people who use our communities on codidact.com can chat informally, and our development server...

"Community" can mean several things, including "dev community" in the OSS world, so a few extra words about what that server is for seem helpful.

Reply:
That's a bit verbose, but making such a clarification is probably a good idea.